pkg/data/sets.go

```diff
@@ -0,0 +1,29 @@
+package data
+
+type StringSet map[string]struct{}
```

Comment:
Provide a string set data structure to avoid needing to linearly scan through arrays to build sets of strings. 

https://github.com/activecm/rita/commit/c4b5b3fa83b503af6da624a9727cbf1bfbaaa45a